docs/content/doc/installation/from-package.en-us.md

```diff
@@ -47,9 +47,52 @@ pacman -S gitea
 
 There is a [Gitea Snap](https://snapcraft.io/gitea) package which follows the latest stable version.
 
-``sh
-snap install gitea
-``
+### This guide was tested with Ubuntu 20.04 and MySQL:
+
+* Install needed Packages and the snap:
+
+```
+sudo snap install gitea
+sudo apt install nginx mariadb-server
+```
+* Secure your mysql Installation
+
+``sudo mysql_secure_installation``
+
+* Create your Database and Database User:
+
+```
+sudo mysql -u root -p
+        CREATE DATABASE gitea;
+        GRANT ALL PRIVILEGES ON gitea.* TO 'gitea'@'localhost' IDENTIFIED BY "StrongPassword";
+        FLUSH PRIVILEGES;
+        QUIT;
+```
+
+* Create nginx Config to pass traffic to port 3000:
```

Comment:
```suggestion
* Create Nginx config to pass traffic to port 3000:
```

Reply:
`nginx` is the correct spelling: http://nginx.org/en/